### DISCUSSION TITLE
Some ideas for import/export

### DISCUSSION THREAD

**Discussion by smitty97** [kenz67/FlightSim-FlightPlanManager#3] [Ideas]
Hi!  just had some ideas that could improve the app.

At launch, and while the app is running, it could monitor a folder for changes (likely Downloads, but changeable) and auto-import any new .pln it finds

And the export folder could be set to the folder that MSFS opens to when you click 'Load' from the map:
for store version this is: %localappdata%l\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState 

Unreleated, but can you also add a 'Notes' column, for anything you'd want to remember about the plan.

**Comment by kenz67** [comment]
Thank you for your comments.   I'll think about the folder monitoring, maybe as an option, I'm not sure how useful that is for the average user.

As for export folder, when you do an import, it captures that folder and will do future imports/exports to that folder as well.   This folder can be different depending on steam vs store and where you have the application installed to, so (as far as I know) there is no way to know for sure what folder to use.

There is a notes column.   Looks like you found a bug that sometimes the scroll bar wasn't working.  New release uploaded to fix that issue

**Reply by smitty97** [reply]
Oh!  I didn't notice that the column of waypoints was called 'notes'.  can there be a new column made?  Also that waypoint column doesn't sort numerically.  And it should probably have a number up front with the total.  i.e.  "5 - 3 waypt, 2 aprt"

That's good that the program remembers the export directory, because MSFS sure doesnt! it always reverts back to the same place.

one more thing :)  can you import the file 'created date' to a field?  i have 300 plans that all say 'imported 2/5/2022' and that isnt very useful!

really love how this is turning out!
